test/JDBC/input/BABEL-EXECUTE_AS_CALLER.sql

```diff
@@ -0,0 +1,97 @@
+-- EXECUTE AS CALLER
```

Comment:
Better as well to change the test into upgrade test format. 
Team is considering deprecating all normal JDBC tests and only keep upgrade tests. Even if there's no dependency needs to be tested, we can still run the test in upgrade scenario (because it would use different settings/configs than normal JDBC env)

Reply:
Fixed in next revision by changing the test to upgrade tests and added them in all version's upgrade schedule.